concepticondata/conceptlists/Dolch-1936-220.tsv

```diff
@@ -64,15 +64,15 @@ Dolch-1936-220-62	62	adverbs	no	1269	NO
 Dolch-1936-220-63	63	adverbs	not	1240	NOT
 Dolch-1936-220-64	64	adverbs	now	1376	NOW
 Dolch-1936-220-65	65	adverbs	off		
-Dolch-1936-220-66	66	adverbs	once		
+Dolch-1936-220-66	66	adverbs	once	2914	ONCE (IN THE PAST)
 Dolch-1936-220-67	67	adverbs	only	1580	ONLY
 Dolch-1936-220-68	68	adverbs	out		
-Dolch-1936-220-69	69	adverbs	so		
+Dolch-1936-220-69	69	adverbs	so	2336	OF THIS KIND (SUCH)
```

Comment:
The linking seems reasonable, not sure whether the concept definition is fitting.

Reply:
It's fine.